### ISSUE TITLE
ssl certificate error for data download

### ISSUE THREAD

**Issue by ijmarshall** [spacy-io/spaCy#488]
I am getting a certificate error when attempting to download the model data.

This happens with fresh installs on Mac OS X with both Anaconda Python 2 and 3 with spacy installed with either conda (version 0.100.7) or pip (version 0.101.0).

Spacy installs correctly, but when I run `python -m spacy.en.download`, the following error occurs.

```
Traceback (most recent call last):
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/site-packages/spacy/en/download.py", line 13, in <module>
    plac.call(main)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/site-packages/plac_core.py", line 328, in call
    cmd, result = parser.consume(arglist)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/site-packages/plac_core.py", line 207, in consume
    return cmd, self.func(*(args + varargs + extraopts), **kwargs)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/site-packages/spacy/en/download.py", line 9, in main
    download('en', force)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/site-packages/spacy/download.py", line 24, in download
    package = sputnik.install(about.__title__, about.__version__, about.__models__[lang])
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/site-packages/sputnik-0.9.3-py2.7.egg/sputnik/__init__.py", line 37, in install
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/site-packages/sputnik-0.9.3-py2.7.egg/sputnik/index.py", line 84, in update
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/site-packages/sputnik-0.9.3-py2.7.egg/sputnik/session.py", line 43, in open
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/urllib2.py", line 429, in open
    response = self._open(req, data)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/urllib2.py", line 447, in _open
    '_open', req)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/urllib2.py", line 1241, in https_open
    context=self._context)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/urllib2.py", line 1195, in do_open
    h.request(req.get_method(), req.get_selector(), req.data, headers)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/httplib.py", line 1057, in request
    self._send_request(method, url, body, headers)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/httplib.py", line 1097, in _send_request
    self.endheaders(body)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/httplib.py", line 1053, in endheaders
    self._send_output(message_body)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/httplib.py", line 897, in _send_output
    self.send(msg)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/httplib.py", line 859, in send
    self.connect()
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/httplib.py", line 1278, in connect
    server_hostname=server_hostname)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/ssl.py", line 353, in wrap_socket
    _context=self)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/ssl.py", line 601, in __init__
    self.do_handshake()
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/ssl.py", line 838, in do_handshake
    match_hostname(self.getpeercert(), self.server_hostname)
  File "/Users/iain/anaconda3/envs/py2/lib/python2.7/ssl.py", line 272, in match_hostname
    % (hostname, ', '.join(map(repr, dnsnames))))
ssl.CertificateError: hostname 'index.spacy.io' doesn't match either of '*.s3.amazonaws.com', 's3.amazonaws.com'
```

**Comment by ijmarshall** [comment]
though almost certainly connected with Issue [485](https://github.com/spacy-io/spaCy/issues/485)

**Comment by honnibal** [comment]
I'd say this is a side-effect of 485, yes --- closing this.

**Comment by ines** [comment]
Should hopefully be fixed now – just posted this in #507:

> We've been moving the site over to new servers and fixing the SSL certificate to finally address the model downloading issues. The problem was that @honnibal was using CloudFlare's "flexible" SSL option. We now got rid of that and installed a certificate from [Let's Encrypt](https://letsencrypt.org) (which is great, btw!)
> 
> Make sure to flush your DNS cache before you reload and try again:
> In Chrome: `chrome://net-internals/#dns`
> On Mac / OSX 10.11+: `sudo dscacheutil -flushcache`
> 
> Alternatively, you can also use [Google Public DNS](https://developers.google.com/speed/public-dns/), which has already updated. That's also what we used for debugging internally for the past few hours. Unfortunately, the DNS is still propagating and seems to be taking forever (see [here](http://viewdns.info/propagation/?domain=spacy.io) for the current status).

If you're still having problems, let us know over at #507.
